app/controllers/AssessmentController.scala

```diff
@@ -98,21 +94,19 @@ class AssessmentController @Inject() (
         } yield {
 
           val exemptions = categorisationInfo.categoryAssessments(index).exemptions
-          val form       = formProvider(exemptions.map(_.id))
+          val form       = formProvider(exemptions.map(_.id), exemptions.size)
 
           form
             .bindFromRequest()
             .fold(
               formWithErrors => {
-                val radioOptions = AssessmentAnswer.radioOptions(exemptions)
-                val viewModel    = AssessmentViewModel(
-                  commodityCode = categorisationInfo.commodityCode,
-                  numberOfThisAssessment = index + 1,
-                  numberOfAssessments = categorisationInfo.categoryAssessments.size,
-                  radioOptions = radioOptions
-                )
+                val listItems = exemptions.map { exemption =>
```

Comment:
this function could maybe be part of the exemption class? Not too bothered either way though just saw the code duplication

def itemDescription = code + " - " +  description

exemptions.map(_.itemDescription)

Reply:
Yep! Moved to CategoryAssessment.scala as we need it for all exemptions, not just a single one.